packages/sdk-core/src/model/http/model/submit/request/BacktraceSubmitStackFrame.ts

```diff
@@ -0,0 +1,66 @@
+interface BaseBacktraceSubmitStackFrame {
```

Comment:
I really don't like the name. Can we rename it to the BacktraceSubmitStackFrameBase instead?

Reply:
renamed to `BacktraceSubmitBaseStackFrame` to follow previous suggestions. `base` is an adjective here, not a noun, so I think it shouldn't be at the end.